diff --git a/include/gazebo_geotagged_images_plugin.h b/include/gazebo_geotagged_images_plugin.h
index 1e2a2c3..c358ed1 100644
--- a/include/gazebo_geotagged_images_plugin.h
+++ b/include/gazebo_geotagged_images_plugin.h
@@ -18,7 +18,7 @@
 
 #include <string>
 #include <opencv2/opencv.hpp>
-#include <mavlink/v2.0/common/mavlink.h>
+#include <mavlink/v2.0/yuneec/mavlink.h>
 #include <gazebo/common/Plugin.hh>
 #include <gazebo/sensors/CameraSensor.hh>
 #include <gazebo/gazebo.hh>
