diff --git a/include/gazebo_geotagged_images_plugin.h b/include/gazebo_geotagged_images_plugin.h
index 6cd6a0c..d0d8f29 100644
--- a/include/gazebo_geotagged_images_plugin.h
+++ b/include/gazebo_geotagged_images_plugin.h
@@ -17,7 +17,7 @@
 #pragma once

 #include <string>
-#include <mavlink/v2.0/common/mavlink.h>
+#include <mavlink/v2.0/yuneec/mavlink.h>
 #include <gazebo/common/Plugin.hh>
 #include <gazebo/sensors/CameraSensor.hh>
 #include <gazebo/gazebo.hh>
